src/components/SQForm/SQFormHelperText.tsx

```diff
@@ -36,26 +35,27 @@ const helperStateMap = {
 
 export type SQFormHelperTextProps = {
   isFailedState?: boolean;
+  isValidState?: boolean;
   errorText?: string;
   failText?: string;
   validText?: string;
 };
 
 function SQFormHelperText({
   isFailedState = false,
+  isValidState = false,
   errorText = 'There is an error in the form',
   failText = 'Cannot proceed',
   validText = 'All fields completed',
 }: SQFormHelperTextProps): JSX.Element {
   const classes = useStyles();
-  const {isValid} = useFormikContext();
   const {fail, error, valid} = helperStateMap;
 
   const getHelperTextType = () => {
     switch (true) {
       case isFailedState:
         return 'fail';
-      case isValid:
+      case isValidState:
```

Comment:
I think we can still use `isValid` here to ensure all our current components still work the way they're supposed to.

Still accept `isValidState` in as a prop to `SQFormHelperText` but make it optional (`isValidState?: boolean`) with no default value so that it could be undefined. Then we can use `isValidState` if it exists and give it priority. If it doesn't exist, fall back to what we had before (`isValid`). 

This route gives consumers more control over the text if they want it, but if they don't it'll work the way that it has in the past.